specification/peering/resource-manager/readme.md

```diff
@@ -0,0 +1,64 @@
+# Peering
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Peering.
+
+
+
+---
+## Getting Started 
+To build the SDK for Peering, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the Peering API.
+
+``` yaml
+openapi-type: arm
+tag: package-2019-03-01-preview
+```
+
+### Tag: package-2019-03-01-preview
+
+These settings apply only when `--tag=package-2019-03-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-03-01-preview'
+input-file:
+- Microsoft.Peering/preview/2019-03-01-preview/peering.json
+```
+# Code Generation
+
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-go
```

Comment:
please add lines for all languages with their corresponding readme files.

Reply:
updated in recent commits.